quintilesims/layer0
# Issue 281

**What does this pull request do?**
Fixes a minor typo: 'data.layer0_api.vpc_id' -> 'data.layer0_api.config.vpc_id'


**How should this be tested?**

1. Execute `mkdocs serve from the docs-src directory.
2. Confirm that the typo is fixed by navigating to the API Data Source sub-section of Layer0 Terraform Plugin section.


**Checklist**


closes #281
